fvm/errors/errors.go

```diff
@@ -134,21 +138,36 @@ func HasErrorCode(err error, code ErrorCode) bool {
 
 // This recursively unwraps the error and returns first CodedError that matches
 // the specified error code.
-func Find(err error, code ErrorCode) CodedError {
-	if err == nil {
+func Find(originalErr error, code ErrorCode) CodedError {
+	if originalErr == nil {
 		return nil
 	}
 
-	var coded CodedError
-	if !As(err, &coded) {
+	var unwrappable Unwrappable
+	if !As(originalErr, &unwrappable) {
 		return nil
 	}
 
-	if coded.Code() == code {
+	coded, ok := unwrappable.(CodedError)
+	if ok && coded.Code() == code {
 		return coded
 	}
 
-	return Find(coded.Unwrap(), code)
+	// NOTE: we need to special case multierror.Error since As() will only
+	// inspect the first error within multierror.Error.
+	errors, ok := unwrappable.(*multierror.Error)
+	if !ok {
+		return Find(unwrappable.Unwrap(), code)
+	}
+
+	for _, internalErr := range errors.Errors {
+		coded = Find(internalErr, code)
```

Comment:
internalErr sounds like a "bug"

```suggestion
	for _, innerError := range errors.Errors {
		coded = Find(innerError, code)
```